app/global-styles.scss

```diff
@@ -154,4 +154,12 @@ body {
 // fixes for carousel slider issue
 .carousel.carousel-slider .slider {
     overflow: initial;
-}
\ No newline at end of file
+}
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active  {
+    -webkit-box-shadow: 0 0 0 30px white inset !important;
```

Comment:
no need to use `-webkit-*` only for specific browser. Direct styling should be enough since we cover all browser